app/models/review.rb

```diff
@@ -7,6 +7,13 @@ class Review < ActiveRecord::Base
   belongs_to :person
   belongs_to :article_revision
 
+RATING_MAPPING = {
```

Comment:
Po przemyśleniu tego wszystkiego wychodzi na to, że w ogóle nie powinno być tego nowego pola `rating`.
Trzeba rozbić wartość `do poprawy` na dwie nowe: `niewielkie poprawki => :minor_review` oraz `istotne poprawki => :major_review`.